i18n/dnd5e/gl/dnd5e.json

```diff
@@ -0,0 +1,758 @@
+  "DND5E.ItemConsumableUsage": "Uso de Consumíbel",
+  "DND5E.ItemConsumableDetails": "Detalles do Consumíbel",
+  "DND5E.ItemConsumableStatus": "Status do Consumíbel",
+  "DND5E.ItemConsumableType": "Tipo do Consumíbel",
+  "DND5E.ItemContainerCapacity": "Capacidade",
+  "DND5E.ItemContainerCapacityItems": "Itens",
+  "DND5E.ItemContainerCapacityType": "Tipo de Capacidade",
+  "DND5E.ItemContainerCapacityWeight": "Peso",
+  "DND5E.ItemContainerDetails": "Detalles do Recipiente",
+  "DND5E.ItemContainerProperties": "Propriedade do Recipiente",
+  "DND5E.ItemContainerWeightless": "Contido sen peso",
+  "DND5E.ItemCreate": "Crear Item",
+  "DND5E.ItemDelete": "Excluir Item",
+  "DND5E.ItemDestroyEmpty": "Destruir ao Vaciar",
+  "DND5E.ItemEdit": "Editar Item",
+  "DND5E.ItemEquipmentAction": "Acción do Equipamento",
+  "DND5E.ItemEquipmentDetails": "Detalles do Equipamento",
+  "DND5E.ItemEquipmentDexMod": "Modificador de Destreza Máximo",
+  "DND5E.ItemEquipmentStatus": "Status do Equipamento",
+  "DND5E.ItemEquipmentStealthDisav": "Impón Desvantaxe en Furtividade",
+  "DND5E.ItemEquipmentType": "Tipo de Equipamento",
+  "DND5E.ItemEquipmentUsage": "Uso de Equipamento",
+  "DND5E.ItemName": "Nome do Item",
+  "DND5E.ItemNew": "Novo(a) {type}",
+  "DND5E.ItemRechargeCheck": "{name} - Proba de recarga",
+  "DND5E.ItemRechargeFailure": "falla!",
+  "DND5E.ItemRechargeSuccess": "éxito!",
+  "DND5E.ItemRequiredStr": "Forza Necesaria",
+  "DND5E.ItemToolProficiency": "Proficiencia con Ferramenta",
+  "DND5E.ItemWeaponAttack": "Ataque da Arma",
+  "DND5E.ItemWeaponDetails": "Detalles da Arma",
+  "DND5E.ItemWeaponProperties": "Propriedade da Arma",
+  "DND5E.ItemWeaponStatus": "Status da Arma",
+  "DND5E.ItemWeaponType": "Tipo da Arma",
+  "DND5E.ItemWeaponUsage": "Uso da Arma",
+  "DND5E.JackOfAllTrades": "Pau para Toda Obra",
+  "DND5E.LairAct": "Usa Acción de Cubil",
+  "DND5E.LairActionLabel": "Acción de Cubil",
+  "DND5E.Languages": "Idiomas",
+  "DND5E.LanguagesAarakocra": "Aarakocra",
+  "DND5E.LanguagesAbyssal": "Abisal",
+  "DND5E.LanguagesAquan": "Aquan",
+  "DND5E.LanguagesAuran": "Auran",
+  "DND5E.LanguagesCelestial": "Celestial",
+  "DND5E.LanguagesCommon": "Común",
+  "DND5E.LanguagesDeepSpeech": "Dialecto Escuro",
+  "DND5E.LanguagesDraconic": "Draconico",
+  "DND5E.LanguagesDruidic": "Druídico",
+  "DND5E.LanguagesDwarvish": "Anano",
+  "DND5E.LanguagesElvish": "Élfico",
+  "DND5E.LanguagesGiant": "Xigante",
+  "DND5E.LanguagesGith": "Gith",
+  "DND5E.LanguagesGnoll": "Gnoll",
+  "DND5E.LanguagesGnomish": "Gnómico",
+  "DND5E.LanguagesGoblin": "Goblin",
+  "DND5E.LanguagesHalfling": "Mediano",
+  "DND5E.LanguagesIgnan": "Ignan",
+  "DND5E.LanguagesInfernal": "Infernal",
+  "DND5E.LanguagesOrc": "Orco",
+  "DND5E.LanguagesPrimordial": "Primordial",
+  "DND5E.LanguagesSylvan": "Silvestre",
+  "DND5E.LanguagesTerran": "Terran",
+  "DND5E.LanguagesThievesCant": "Barallete de Ladróns",
+  "DND5E.LanguagesUndercommon": "Subcomún",
```

Comment:
A verdade é que aquí me gusta máis o castelán "Infracomún", pero é cuestión de gustos 😄